src/modules/windowwalker/app/Window Walker/MainWindow.xaml.cs

```diff
@@ -51,6 +51,9 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
 
             UpdateDisplayedVersionNumber();
 
+            Left = (SystemParameters.WorkArea.Width - ActualWidth) / 2.0;

```

Comment:
Nice 👍 . Consider moving this calculation to be on every display since people can plugin/remove/change DPI on their monitors?

Reply:
my thought here is we'll migrate the code over to Launcher so i'm fine with this being here for a while on a single screen